### ISSUE TITLE
Part Number (for journalArticle)

### ISSUE THREAD

**Issue by rmzelle** [citation-style-language/zotero-bits#12]
A new field, "Part Number"/**partNumber**, should be added to journalArticle, and mapped to **number** in CSL.

https://www.zotero.org/trac/ticket/1368

**Comment by rmzelle** [comment]
@adam3smith, do you know if CMoS says anything about articles published in parts?

(I encountered this in my own writing, but I was able to just add the part number and subtitle to the "title" field:

Malic acid accumulation by Aspergillus flavus. I. Biochemical aspects of acid biosynthesis (https://doi.org/10.1007/BF00250501)
Malic acid accumulation by Aspergillus flavus. II. Crystals and hair-like processes formed by A. flavus in a l-malic acid production medium (https://doi.org/10.1007/BF00250502)
Malic acid accumulation by Aspergillus flavus. III. 13C NMR and isoenzyme analyses (https://doi.org/10.1007/BF00264008)
)

**Comment by adam3smith** [comment]
Yes. 14.189, here in its entirety (italics ommitted):

Articles published in parts over two or more issues may be listed separately or in the same entry, depending on whether the part or the whole is cited.

    68. George C. Brown, ed., “A Swedish Traveler in Early Wisconsin: The Observations of Fredrika Bremer,” pt. 1, Wisconsin Magazine of History 61 (Summer 1978): 312.
    69. Ibid., pt. 2, Wisconsin Magazine of History 62 (Autumn 1978): 50.

    Brown, George C., ed. “A Swedish Traveler in Early Wisconsin: The Observations of Fredrika Bremer.” Pts. 1 and 2. Wisconsin Magazine of History 61 (Summer 1978): 300–318; 62 (Autumn 1978): 41–56.

**Comment by rmzelle** [comment]
Great, thanks. @adam3smith, do you also agree that we'll probably want to treat this information separately from "section", per above at https://github.com/citation-style-language/zotero-bits/issues/12#issuecomment-777628, and that we'll need a new CSL variable (e.g. "part-number" or just "part")?

**Comment by adam3smith** [comment]
I agree it needs to be separate from section, yes (section is information about the container, part refers to the article; also see Gracile's example above). I do think it's a bit unfortunate to include these two variables just for two relatively rare cases, though. 
Can we think of any way in which they can cover something broader?

**Comment by fbennett** [comment]
On Mon, Feb 13, 2017 at 3:04 AM, Sebastian Karcher <notifications@github.com
> wrote:

> I agree it needs to be separate from section, yes (section is information
> about the container, part refers to the article; also see Gracile's example
> above). I do think it's a bit unfortunate to include these two variables
> just for two relatively rare cases, though.
>
IIRC the Zotero variables "series" and "seriesTitle" are meant to have
different meanings (I forget which is which - one is a series enclosing the
container?, the other a series across container volumes?). It think they
both map to collection-title at the moment. If a new part-number is cast,
should a companion *-title also be cast, and associated with series (or
seriesTitle)?

**Comment by gracile-fr** [comment]
@fbennett : https://forums.zotero.org/discussion/comment/94959/#Comment_94959 (meaning of "series", "seriesTitle", "seriesText"). See also #21

**Comment by bwiernik** [comment]
Is the consensus that both `part-number` and `part-title` be created?

**Comment by denismaier** [comment]
That's accepted, right? Would be also nice to have this for encyclopedia articles.

**Comment by bwiernik** [comment]
Adding `zotero` tag because this will be covered by https://github.com/citation-style-language/schema/issues/179

**Comment by dstillman** [comment]
Sorry, where did we land with this? `partNumber` and `partTitle` for `journalArticle`, mapped to CSL `part-number` and `part-title`?

**Comment by bwiernik** [comment]
That's correct. 

The CSL 1.0.2 variables are listed here if that is helpful https://github.com/citation-style-language/documentation/blob/master/specification.rst

I can also take a look at a changelist
